plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/organize/imports/ImportProvidedElementLabelprovider.java

```diff
@@ -12,25 +12,30 @@
 
 import org.eclipse.jface.viewers.ILabelProvider;
 import org.eclipse.jface.viewers.ILabelProviderListener;
+import org.eclipse.n4js.n4JS.ImportDeclaration;
+import org.eclipse.n4js.organize.imports.ImportProvidedElement;
+import org.eclipse.n4js.projectModel.IN4JSCore;
+import org.eclipse.n4js.projectModel.IN4JSProject;
+import org.eclipse.n4js.ts.types.TExportableElement;
+import org.eclipse.n4js.ts.types.TModule;
+import org.eclipse.n4js.ui.labeling.N4JSLabelProvider;
 import org.eclipse.swt.graphics.Image;
 import org.eclipse.xtext.naming.IQualifiedNameConverter;
 import org.eclipse.xtext.resource.IEObjectDescription;
 
 import com.google.inject.Inject;
 
-import org.eclipse.n4js.n4JS.ImportDeclaration;
-import org.eclipse.n4js.organize.imports.ImportProvidedElement;
-import org.eclipse.n4js.ts.types.TModule;
-import org.eclipse.n4js.ui.labeling.N4JSLabelProvider;
-
 /**
+ * Provides labels used in imports disambiguation dialog.
```

Comment:
As discussed, consider reuse `N4JSLabelProvider` to have beautiful icons in the list of possible candidates in the ambitious dialog.